doc/user/install_guide.md

```diff
@@ -1,36 +1,38 @@
-# Operation Guide
+# Installation guide
 
-## Setup RBAC
+## Set up RBAC
 
-Set up basic [RBAC rules](./rbac.md) for etcd operator:
+Set up basic [RBAC rules][rbac-rules] for etcd-operator:
```

Comment:
i wonder what's rule for using `-`.
should we use `etcd operator` or ` etcd-operator` for the doc?
I think either one is fine. but we need to keep everything consistent.

Reply:
Sorry that I merged it fast.
I'm fine with both actually.
Is it OK for both to co-exist?

Reply:
Or maybe we should change "etcd-operator" here back to "etcd operator"?

Reply:
@zbwright 
Can you change "etcd-operator" in this doc back to "etcd operator"? Thanks!

Reply:
i think we just need to stick with one.

Reply:
@zbwright what's the best practice in this case?

Reply:
best practice in this instance is for all three OCS operator docs to be consistent - either they are 'vault-operator, etcd-operator, and prometheus-operator', or they are 'vault operator, etcd operator, and prometheus operator'. Product should pick one. If we drop the hyphen, we should probably call it 'the etcd operator', rather than 'etcd-operator'. I'm leaving all this fine tuning for a later date. we'll drop the hyphen in all these docs for now.

Reply:
sounds good!

Reply:
@fanminshi I don't see a vault operator anywhere in coreos. Is it in any other repository?

Reply:
@raoofm 
Vault operator is private.
@zbwright is our internal technical editor so she needed to consider all products as a whole.

Reply:
@hongchaodeng thank you